pipelines/broad/arrays/single_sample/Arrays.wdl

```diff
@@ -489,33 +489,33 @@ workflow Arrays {
   }
 
   output {
-    String ChipWellBarcodeOutput = IlluminaGenotypingArray.chip_well_barcode_output
-    Int AnalysisVersionNumberOutput = IlluminaGenotypingArray.analysis_version_number_output
-    File GtcFile = IlluminaGenotypingArray.gtc
-    File RedIdatMd5CloudPath = IlluminaGenotypingArray.red_idat_md5_cloud_path
-    File GreenIdatMd5CloudPath = IlluminaGenotypingArray.green_idat_md5_cloud_path
-    File? OutputVcfMd5CloudPath = IlluminaGenotypingArray.output_vcf_md5_cloud_path
-    File? OutputVcfFile = IlluminaGenotypingArray.output_vcf
-    File? OutputVcfIndexFile = IlluminaGenotypingArray.output_vcf_index
-    File? BafRegressMetricsFile = CreateBafRegressMetricsFile.output_metrics_file
+    String chip_well_barcode_output = IlluminaGenotypingArray.chip_well_barcode_output
+    Int analysis_version_number_output = IlluminaGenotypingArray.analysis_version_number_output
+    File gtc_file = IlluminaGenotypingArray.gtc
+    File red_idat_md5_cloud_path = IlluminaGenotypingArray.red_idat_md5_cloud_path
+    File green_idat_md5_cloud_path = IlluminaGenotypingArray.green_idat_md5_cloud_path
+    File? output_vcf_md5_cloud_path = IlluminaGenotypingArray.output_vcf_md5_cloud_path
+    File? output_vcf = IlluminaGenotypingArray.output_vcf
+    File? output_vcf_index = IlluminaGenotypingArray.output_vcf_index
+    File? baf_regress_metrics_file = CreateBafRegressMetricsFile.output_metrics_file
     File? ContaminationMetricsFile = IlluminaGenotypingArray.contamination_metrics
```

Comment:
should `ContaminationMetricsFile` have been updated to python naming convention as well?